tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -827,49 +835,65 @@ export class WebGPUBackend extends KernelBackend {
       this.makeUniforms(programUniform)
     ];
```

Comment:
I seems that in `ENGINE_COMPILE_ONLY` mode, the `bindings`/ `programUniform` are not necessary. They all can be skipped. So it will be better to move them into `recordAndSubmit`.